clients/cli/src/unwrap.rs

```diff
@@ -0,0 +1,231 @@
+use {
+    crate::{
+        common::{
+            get_account_owner, get_mint, parse_pubkey, parse_token_program, process_transaction,
+        },
+        config::Config,
+        output::{format_output, println_display},
+        CommandResult,
+    },
+    clap::{value_parser, Args},
+    serde_derive::{Deserialize, Serialize},
+    serde_with::{serde_as, DisplayFromStr},
+    solana_clap_v3_utils::{
+        input_parsers::signer::{SignerSource, SignerSourceParserBuilder},
+        keypair::signer_from_source,
+    },
+    solana_cli_output::{display::writeln_name_value, QuietDisplay, VerboseDisplay},
+    solana_hash::Hash,
+    solana_pubkey::Pubkey,
+    solana_remote_wallet::remote_wallet::RemoteWalletManager,
+    solana_signature::Signature,
+    solana_signer::Signer,
+    solana_transaction::Transaction,
+    spl_token_wrap::{get_wrapped_mint_address, get_wrapped_mint_authority, instruction::unwrap},
+    std::{
+        fmt::{Display, Formatter},
+        rc::Rc,
+        sync::Arc,
+    },
+};
+
+#[derive(Clone, Debug, Args)]
+pub struct UnwrapArgs {
+    /// The address of the wrapped token account to unwrap from
+    #[clap(value_parser = parse_pubkey)]
+    pub wrapped_token_account: Pubkey,
+
+    /// The address of the escrow account holding the unwrapped tokens
+    #[clap(value_parser = parse_pubkey)]
+    pub escrow_account: Pubkey,
+
+    /// The address of the token account to receive the unwrapped tokens
+    #[clap(value_parser = parse_pubkey)]
+    pub unwrapped_token_recipient: Pubkey,
```

Comment:
This could default to the ATA for the payer or the transfer authority, but maybe that's too much done implicitly

Reply:
Fine by me!